src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -52,5 +52,6 @@ internal static class CompletionControllerOptions
     {
         public static readonly Option<bool> FilterOutOfScopeLocals = new Option<bool>(nameof(CompletionControllerOptions), nameof(FilterOutOfScopeLocals), defaultValue: true);
         public static readonly Option<bool> ShowXmlDocCommentCompletion = new Option<bool>(nameof(CompletionControllerOptions), nameof(ShowXmlDocCommentCompletion), defaultValue: true);
+        public static readonly Option<bool> TextEditorSnipppetsAreActive = new Option<bool>(nameof(CompletionControllerOptions), nameof(TextEditorSnipppetsAreActive), defaultValue: false);
```

Comment:
The name here doesn't really seem like it fits. It's not that "snippets are active", it's more that the caret is in an active snippet field. Maybe something like, `TextEditorSnippetFieldIsActive`?